clang/include/clang/StaticAnalyzer/Checkers/Checkers.td

```diff
@@ -1703,6 +1703,12 @@ def TaintedAllocChecker: Checker<"TaintedAlloc">,
   Dependencies<[DynamicMemoryModeling, TaintPropagationChecker]>,
   Documentation<HasDocumentation>;
 
+def TaintedDivChecker: Checker<"TaintedDiv">,
+  HelpText<"Check for divisions, where the denominator "
+           "might be 0 as it is a tainted (attacker controlled) value.">,
```

Comment:
```suggestion
  HelpText<"Check for divisions where the denominator is tainted "
           "(attacker controlled) and might be 0.">,
```
Again, swapping the order clearly clarifies that taintedness is not covered by the "might". (Your wording was also understandable, but more complicated.)